packages/AutoTDD-Core.package/ATDDTestingEntity.class/instance/asStringOrText.st

```diff
@@ -0,0 +1,4 @@
+accessing
+asStringOrText
+
+^ self asString asText addAttribute: (TextColor color: self statusColor)
```

Comment:
Why do you call `asString` and then `asText`?

Reply:
asString makes a string for the testing entity (possibly defined by sublass), asText gives the possibility of adding color